asus/zephyrus/ga402/default.nix

```diff
@@ -0,0 +1,20 @@
+{ ... }:
+
+{
+  imports = [
+    ../../../common/cpu/amd
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/gpu/amd
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/acpi_call.nix
+    ../../../common/pc/laptop/ssd
+  ];
+
+  services = {
+    asusd.enable = true;
+  };
+
+  boot = {
+    kernelParams = [ "pcie_aspm.policy=powersupersave" "acpi.prefer_microsoft_dsm_guid=1" ];
```

Comment:
The second option shouldn't be necessary with recent firmware updates.

Reply:
Which firmware ? I still saw the problem with BIOS 315. I'll try 316 then

Reply:
I'm unable to reproduce the issue on 316 or 501.U4.

Reply:
ok going to upgrade then I'll remove the flag.

I saw a difference in term of power consumption with the first but unsure what is the drawback of using it.